cfgov/v1/models/learn_page.py

```diff
@@ -106,6 +107,47 @@ class DocumentDetailPage(AbstractFilterPage):
     template = 'document-detail/index.html'
 
 
+class StoryPage(AbstractFilterPage):
+    video_url = models.URLField("YouTube URL", blank=True,
+                                help_text="Format: https://www.youtube.com/embed/video_id. "
+                                          "It can be obtained by clicking on Share > Embed on YouTube.",
+                                validators=[RegexValidator(
```

Comment:
I'm intrigued. But, given the specificity required for the regex passed to it, what would be the advantage of using URLValidator?